functions/controllers/post/getPostById.js

```diff
@@ -0,0 +1,59 @@
+const { BAD_REQUEST, OK, INTERNAL_SERVER_ERROR } = require('http-status-codes');
+const { message, status } = require('../../util/constants');
+
+const { error, success } = status;
+const { somethingWentWrong } = message;
+
+const postMessage = async (req, res, Post) => {
+    await Post;
+    res.status(OK).json({ data: Post, status: success });
+};
+
+// eslint-disable-next-line max-lines-per-function
```

Comment:
remove disable

Reply:
Can't need to fetch both comments and likes for that post and it would exceed line